includes/config.h

```diff
@@ -36,19 +36,19 @@
 #define PHW_FAULHABER 2
 
 /// Defines which preset to use for controlling pressure
-#define PNEUMATIC_HARDWARE_VERSION PHW_CHU
+#define PNEUMATIC_HARDWARE_VERSION PHW_FAULHABER
 
 // Available valves
 #define VT_SERVO_V1 0
 #define VT_EMERSON_ASCO 1
 #define VT_FAULHABER 2
 
 // Defines which valves are fitted
-#define VALVE_TYPE VT_SERVO_V1
+#define VALVE_TYPE VT_FAULHABER
```

Comment:
So the VT_FAULHABERis now the default valve type?

Reply:
Yes !